samply/src/main.rs

```diff
@@ -482,6 +519,13 @@ impl ImportArgs {
             let name = self.file.file_name().unwrap_or(self.file.as_os_str());
             name.to_string_lossy().into()
         };
+        cfg_if::cfg_if! {
+            if #[cfg(target_os = "windows")] {
+                let time_range = self.time_range;
```

Comment:
These braces are lies!

Can you make it a `let mut time_range = None;` at the function scope and then mutate it inside the braces? I find it too confusing otherwise.

Reply:
Yes :) I hate this pattern, would be nice to do something.. nicer.

Reply:
Oh and then Rust is going to complain that the mut is unused on non-Windows... you just can't win

Reply:
I think this works:
```rust
        let time_range = {
            cfg_if::cfg_if! {
                if #[cfg(target_os = "windows")] {
                    self.time_range
                } else {
                    None
                }
            }
        };
```

Reply:
Nope; that's what I tried first. `Attributes on expressions are experimental.`

Reply:
I'll just write out the expressions for now in the struct initialization. I think in the future we can just add `WindowsImportArgs` or similar that can get flattened in or similar.

Reply:
It doesn't matter now, but it does work if you include the extra braces, like I showed, since that converts it into a statement. Omitting those will give the error you saw.